modules/edit-modes/src/lib/modify-mode.ts

```diff
@@ -124,8 +124,7 @@ export class ModifyMode extends GeoJsonEditMode {
         'Editing 3D point but modeConfig.viewport not provided. Falling back to 2D logic.'
       );
     }
-
-    return nearestPointOnLine(line, inPoint);
+    return nearestPointOnLine(line, inPoint, viewport);
```

Comment:
I know it was already an issue, but can we rename one of the `nearestPointOnLine` functions? It is weird to have a `this.nearestPointOnLine` and `nearestPointOnLine` functions in the same file. Perhaps just `this.getNearestPoint` and `nearestPointOnLine`.

Reply:
sure, can do on this.